### ISSUE TITLE
Add Scroll to LI.FI/Jumper Multi Chain Bridge 

### ISSUE THREAD

**Issue by Esther111111** [scroll-tech/contribute-to-scroll#874]
### Name of the project

LI.FI/Jumper

### Project type

Tool

### Site of the project

https://jumper.exchange/

### Github repository of the project

https://github.com/lifinance/

### Networks

Scroll Mainnet

### Additional context

Li.fi/ jumper cross chain bridge helps in the movement and sourcing assets across other Blockchain ecosystem. 

It also have split transactions and fall back mechanics in the sense of faster, cheaper and safer route any time. 

I would love to see Scroll integration with them. I noticed more users are using jumper exchange because they can move and swap liquidity across 20+ chains yet Scroll is not found there. Least I forget while using li.fi/jumper exchange, there is refuel option to get gas on a particular chain, instead of withdraw from a CEX.

In conclusion I would love to see Scroll on Jumper exchange.

**Comment by iamodtherealone** [comment]
Great Idea, I just hope the team can look forward to solving this gap.

**Comment by Robinsonwealth** [comment]
Love this idea. Been using Jumper for a while now. Hoping to see it come through.

**Comment by Esther111111** [comment]
> Great Idea, I just hope the team can look forward to solving this gap. 

They will if the devs are seeing this. All projects want improvement

**Comment by Esther111111** [comment]
> Awesome. Love this 

I use jumper exchange frequently and I love to see Scroll there
